manual/src/threelang/web3gw/money.md

```diff
@@ -0,0 +1,80 @@
+# Money
+Is an inter-chains centerilized actor that can do all the money related actions on the supported chains (`tfchain`, `stellar`, `etherum`, `bitcoin`, ...).
+
+## Actions
+
+### Send tokens between accounts
+Send tokens from one account to another account on the same chain.
+- supported sends:
+    - btc on bitcoin -> btc on bitcoin
+    - eth on ethereum -> eth on ethereum
+    - token on ethereum -> token on ethereum
+    - tft on stellar -> tft on stellar
+    - tft on tfchain -> tft on tfchain
+
+- action name: `!!web3gw.money.send`
+- parameters:
+    - `channel`: the chain where the transaction will be done. (`tfchain`, `stellar`, `ethereum`, `bitcoin`, ...)
+    - `currency`: the currency to send (`tft`, `btc`, `eth`, `xlm`, ...). If the channel or the target_channel is `tfchain` then the currency is `tft` by default.
+    - `from`: the source address or twin_id if the channel is `tfchain`
+    - `to`: the destination address or twin_id if the target_channel is `tfchain`
+    - `amount`: the amount to send
+
+- examples:
+    ```md
+    !!web3gw.money.send
+        channel:tfchain
+        from:29
+        to:28
+        currency:tft
+        amount:100
+    ```
+
+### Swap tokens on the same chain
+Swap from token to another token on the same chain.
+- supported swaps:
+    - eth on ethereum -> tft on ethereum
+    - tft on ethereum -> eth on ethereum
+    - tft on stellar -> xlm on stellar
+
```

Comment:
Also xlm on stellar -> tft on stellar

Reply:
actually i can't find calls in stellar module for swapping.
the one i could find here is for transferring tfts from tfchain to stellar account
https://github.com/threefoldtech/web3_proxy/blob/ade79f396b2d85ef0049abbbbdfc19eeee128232/lib/tfchain/tfchain.v#L321

are we only support swapping to and from eth?

Reply:
Ah right. That's because I added it on my branch that is not merged yet. I'll go ahead and work on that before we can merge this one.

Reply:
added